### PR TITLE
Link Secret Santa rules

### PR DESCRIPTION
**Pull request by exu3**
Alternative version of #278. 
This PR links the rules on the form and left aligns the form heading.

Screen Shot 2021-12-14 at 6 03 37 PM

thoughts on this @neelr?

### CODE REVIEW

File: pages/santa.js [hunk 1]
```diff
@@ -187,7 +187,14 @@ function Signup() {
   const [values, setValues] = useState({})
   return (
     <Base method="get" target="_blank" action="https://hack.af/santa-signup">
-      <Heading sx={{ color: 'black' }}>Register!</Heading>
+      <Heading sx={{ color: 'black', textAlign: 'left' }}>Register!</Heading>
```
Comment (sampoder) [inline pages/santa.js]:
```suggestion
      <Heading sx={{ color: 'black', textAlign: 'left', mb: 2 }}>Register!</Heading>
```

Could we add a slight margin here?

**Review by sampoder — APPROVED** [review]
nice

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/santa**](https://v3-qld8uuuse.hackclub.dev/santa) |
|:-:|
|<a href="https://v3-qld8uuuse.hackclub.dev/santa">Screenshot of /santa</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/jleqS-4ceKaWxHvi4NoYS-N-WxYydX51dYGVEP5-6ojV6YP_kxxIvhimdGlftCl3wfYfzLjdkdAXMySjzGzu5A.png">(view full size)</a> |







Commit 0dda173538a669623f820b3c6a95f1fc0ea7cb76 (https://v3-qld8uuuse.hackclub.dev).

**Comment by exu3** [comment]
Got approval from Neel via Slack.

**Pull request merged**